lib/index.ts

```diff
@@ -203,7 +215,10 @@ export function validatePackageSize(size: number, errorReporter: (errorMessage:
 async function getComponentsDefinition(
     componentsDefinitionContent: string,
     errorReporter: (errorMessage: string) => void,
-): Promise<{ data: ComponentsDefinition | null, pointers: any }> {
+): Promise<{
+    data: ComponentsDefinition | null;
+    pointers: { [key: string]: { [key: string]: { line: number; column: number; pos: number } } } | null;
```

Comment:
As there are no typings for json-source-map I created my own minimal version.

Reply:
Might be fixed by creating our own typings? See also my other comment.